### PR TITLE
[WIP] yarpframecontroller

### PR DESCRIPTION
**Pull request by aerydna**
new module for controlling two transform frames with a joypad.

### CODE REVIEW

**Comment by aerydna** [comment]
All the current codacy errors are not correlated with the pr modified files. @drdanz is it possible to dismiss them?

File: src/yarpframecontroller/ParamParser.h [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#include <yarp/os/Value.h>
```
Comment (barbalberto) [inline src/yarpframecontroller/ParamParser.h]: Missing copyright statement

File: src/yarpframecontroller/yarpframecontroller.cpp [hunk 2]
```diff
@@ -0,0 +1,199 @@
+#include <yarp/dev/IJoypadController.h>
```
Comment (barbalberto) [inline src/yarpframecontroller/yarpframecontroller.cpp]: Missing copyright statement

**Comment by aerydna** [comment]
closing for this software to be now in the repository yarp-joypad-tools.

**Pull request closed without merging**